torch/_torch_docs.py

```diff
@@ -2543,17 +2543,22 @@ def merge_dicts(*dicts):
            r"""
 logical_not(input, out=None) -> Tensor
 
-Computes the element-wise logical NOT of the given input tensor. The input tensor must have the bool dtype.
+Computes the element-wise logical NOT of the given input tensor. The output tensor always have the bool dtype. If the
```

Comment:
I don't think this is true -- at least it shouldn't be.  We should support other dtypes for out, we just use bool if no other ones are provided.